src/__tests__/tsconfig.json

```
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "resolveJsonModule": true,
+    "strict": false
```

This just popped into my head out of the blue but should we add a target of like 2018 or later here?

```suggestion
    "strict": false,
    "target": "es2018"
```

---

i think not now but maybe as we add/change tests we'll want it.